content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,301 @@
+---
+{
+    title: "Introduction to Web Accessibility (A11Y)",
+    description: "",
+    published: '2021-05-30T22:12:03.284Z',
+    authors: ['crutchcorn'],
+    tags: ['a11y', 'javascript', 'web'],
+    attached: [],
+    license: 'coderpad',
+    originalLink: 'https://coderpad.io/blog/introduction-to-accessibility/'
+}
+---
+
+If you're coming across this article and haven't heard about "accessibility" (often shortened to "A11Y") before, that's okay. We're all learning at different speeds and come across new things all the time. That said, accessibility is a critical component of any frontend engineer's responsibilities. We implore you to explore what that means, not just in this blog post but beyond with your teams and communities.
+
+First, let's define what "accessibility" is. Accessibility in engineering is ["the process of creating products that are usable by people with the widest possible range of abilities."](https://dl.acm.org/doi/10.1145/2596695.2596719)
+
+You may have vision-impaired or blind users of your app. These users may rely on [screen readers](https://www.afb.org/blindness-and-low-vision/using-technology/assistive-technology-products/screen-readers), which allows users to navigate their computer with their sense of hearing. These methods may be used in tandem with the visual experience for some users or used independently to navigate their computer auditorily.
```

Comment:
Oh, this call out for "impaired" is fantastic. I did a lot of research (and actually asked in the A11Y slack) about the language around the term "people with disabilities" but never actually thought to introspect on the phrasing "impaired" due to a deep misunderstanding of preferred terms. In hindsight, it makes perfect sense, even with my own thoughts and views on ablist language.

Thanks a bunch for catching this!

As for "users" becoming dehumanizing, I understand that to a degree, but am not sure how much I can do in this article without becoming overly verbose. I definitely want to include "people that use your app" a few times, but don't know if this will make it to every instance. FWIW, this problem seems to extend past A11Y as I'd use similar "users" language when talking about other application functionality/design outside of accessibility discussions as well

Reply:
For anyone that might catch this thread in the future, curious about some of the research done:

- [Brown university has a great guide on appropriate terminology](https://www.brown.edu/campus-life/support/accessibility-services/resources-teaching-students-disabilities/appropriate-terminology)
- [Apple uses the term "people with disabilities"](https://www.apple.com/newsroom/2021/05/apple-previews-powerful-software-updates-designed-for-people-with-disabilities/)
- [Conversation in A11Y slack](https://web-a11y.slack.com/archives/C042TSFGN/p1621931053327600)

(I admittedly don't remember the other places I looked, these instances stood out to me)

Reply:
Absolutely!

And I think the only places that really call out to switch user is where I suggested it.